subsys/bluetooth/audio/pacs.c

```diff
@@ -92,9 +92,16 @@ static struct pacs_client {
 
 	/* Pending notification flags */
 	ATOMIC_DEFINE(flags, FLAG_NUM);
-} clients[CONFIG_BT_MAX_PAIRED];
+};
+
+struct pacs {
+	bool registered;
+	struct bt_bap_pacs_register_param param;
+	atomic_t notify_rdy;
+	atomic_t unregistering;
```

Comment:
All these values could be stored as bits, as they are all boolean values, right?

So instead of using 7 octets on storing this information, it could be 7 bits (1 octet). 

Either as atomic flags or boolean bitfields depending on whether we need the atomic functionality